mysite/scripts/snapshot_then_push.sh

```diff
@@ -30,16 +30,37 @@ cd "$PREFIX"
 
 ## Create a path to store the JSON in
 TEMPFILE="$(mktemp --suffix -snapshot.json.gz)"
+TEMPDB = "$(mktemp --suffix -snapshot.db)"
```

Comment:
In theory, be consistent about style -- don't use ` ` around the `=` here. Not a show-stopper.